test/e2e/testing-manifests/storage-csi/external-provisioner/rbac.yaml

```diff
@@ -32,7 +32,7 @@ rules:
   #   verbs: ["get", "list"]
   - apiGroups: [""]
     resources: ["persistentvolumes"]
-    verbs: ["get", "list", "watch", "create", "delete"]
+    verbs: ["get", "list", "watch", "create", "update", "delete"]
```

Comment:
/hold 

util https://github.com/kubernetes-csi/external-provisioner/pull/1155 is merged and the file is synced with this change via `update-hostpath` script.

https://github.com/kubernetes/kubernetes/pull/122489 will sync these files once hostpath v1.13.0 is released.

Reply:
Additional permission can be added when the csi mock driver is being inited, example: https://github.com/carlory/kubernetes/blob/master/test/e2e/storage/drivers/csi.go#L682-L693